.github/workflows/conventional-commit.yml

```diff
@@ -0,0 +1,68 @@
+name: "Lint PR title"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+permissions:
+  pull-requests: write
+
+jobs:
+  main:
+    name: Validate PR titles
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Install pnpm
+        uses: pnpm/action-setup@v3
+        with:
+          version: 9.1.1
+      - name: Generate scopes
+        id: generate_scopes
+        run: |
+          scopes=$(pnpm m ls --json --depth=-1 | grep "name" | cut -d":" -f2 | grep "@novu" | cut -d"/" -f2 | grep "\"" | cut -d"\"" -f1)
+          echo 'SCOPES<<EOF' >> $GITHUB_ENV
+          echo "$scopes" >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
```

Comment:
Enforce auto-updating scopes, to improve future changelog generation to specify the subject of changes.